server/sandbox_run_linux.go

```diff
@@ -670,13 +670,13 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	return resp, nil
 }
 
-func setupShm(podSandboxRunDir, mountLabel string) (shmPath string, err error) {
-	shmPath = filepath.Join(podSandboxRunDir, "shm")
+func setupShm(podSandboxRunDir, mountLabel string) (string, error) {
```

Comment:
Here we are losing documentation implicit in the return value names. If we do this, then we should add a comment for the function.

Reply:
This case is where I would argue that keeping it as is probably fine as we don't have defers and it is a small function.

Reply:
this one actually snuck...